src/commands/foundation/deploy.command.ts

```diff
@@ -79,6 +81,10 @@ export function registerDeployCmd(program: TopLevelCommand) {
 
         const mode = { raw: literalArgs.length ? literalArgs : ["apply"] };
 
+        const foundation = foundationArg ||
```

Comment:
yes, that would make a lot of sense, especially since we want to enforce a consistent order: -> explicit arg -> config -> interactive prompt?

The interactive prompt will also only work in TTY terminals, I think it does check that properly but we should make sure it doesn't break when stdin is not a TTY